Code/Java/Kamenetsky_Formula.java

```diff
@@ -0,0 +1,63 @@
+
+/*
+ * The Kamenetsky Formula is used to calculate the number of digits in the factorial of a number.
+ * Formula:
+ *        number of digits in factorial = log10( ((n/e)^n) * sqrt(2*pi*n))
```

Comment:
can you add a proof to the formula? or add some reference link??

Reply:
made the changes.